database/dbo/Tables/Positions.sql

```diff
@@ -12,7 +12,10 @@ CREATE TABLE [dbo].[Positions] (
     [PositionNameId] UNIQUEIDENTIFIER CONSTRAINT [FK_Positions_PositionNames] FOREIGN KEY ([PositionNameId]) REFERENCES [dbo].[PositionNames] ([Id]),
     [Koulutustaso] NVARCHAR(255),
     [Tyokokemus] NVARCHAR(255),
-    [Lisatiedot] NVARCHAR(255)
+    [Lisatiedot] NVARCHAR(255),
+    [OrgTreeNumero] NVARCHAR(50) NOT NULL,
+    [OrgTreeAlue] NVARCHAR(255) NOT NULL,
+    CONSTRAINT [FK_Positions_OrganizationTree] FOREIGN KEY ([OrgTreeNumero], [OrgTreeAlue]) REFERENCES [dbo].[OrganizationTree] ([Number], [Alue]) -- Composite foreign key
```

Comment:
This might be better if you add a Guid to the OrgTree model and add a foreign key to that Id. Later on we might need to fetch individual organization tree objects, therefore Guid is a bit better.

Reply:
Fair enough, I'll look into making those changes